### PR TITLE
Update Dispatcher.php

### PR DESCRIPTION
**Pull request by vieirasantosn**
remove the switch and the second indentation level.

### CODE REVIEW

**Comment by hannesvdvreken** [comment]
Clean :+1:. I like it.

**Comment by philipobenito** [comment]
:+1:

File: src/Dispatcher.php [hunk 1]
```diff
@@ -56,25 +56,21 @@ public function __construct(ContainerInterface $container, array $routes, array
...
-                break;
-            case FastRoute::FOUND:
-            default:
-                $handler  = (isset($this->routes[$match[1]]['callback'])) ? $this->routes[$match[1]]['callback'] : $match[1];
-                $strategy = $this->routes[$match[1]]['strategy'];
-                $vars     = (array) $match[2];
-                $response = $this->handleFound($handler, $strategy, $vars);
-                break;
+        
+        if ($match[0] == FastRoute::NOT_FOUND) {
```
Comment (philsturgeon) [inline src/Dispatcher.php]: Would `===` be better here?

Comment (vieirasantosn) [inline src/Dispatcher.php]: I like, more clean and following the workflow.

Comment (philipobenito) [inline src/Dispatcher.php]: Yes should be triple =, cheers Pil, missed that, will update before release.

Comment (philsturgeon) [inline src/Dispatcher.php]: Backseat coding like a champ.

Comment (vieirasantosn) [inline src/Dispatcher.php]: Yes, should be ===, i’ll change it.
Ty.

> Em 23/02/2015, à(s) 2:23 PM, Phil Sturgeon <notifications@github.com> escreveu:
> 
> Backseat coding like a champ.

**Pull request merged**